as/thrift.js

```diff
@@ -54,13 +54,10 @@ function TChannelAsThrift(opts) {
         allowIncludeAlias: opts.allowIncludeAlias,
         allowOptionalArguments: opts.allowOptionalArguments,
         defaultAsUndefined: opts.defaultAsUndefined,
+        releaseSources: true,
```

Comment:
maybe add a one line comment on what this flag does?

Reply:
SG